### PR TITLE
Spec work

### PR DESCRIPTION
**Pull request by LeahBineth**

### CODE REVIEW

**Comment by mordechaigellis** [comment]
got it will review

File: database.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+use master
+go
+drop database if exists test
```
Comment (mordechaigellis) [inline database.sql:3]: Please make the db name more specific

Comment (LeahBineth) [reply database.sql:3]: Which name do you want me to use?

Comment (mordechaigellis) [reply database.sql:3]: try testdb

File: database.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+go
+create database tes
```
Comment (mordechaigellis) [inline database.sql:5]: you forgot the letter t

File: database.sql [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+go
```
Comment (mordechaigellis) [inline database.sql:6]: you gont need a go

**Review by mordechaigellis — CHANGES_REQUESTED** [review]

**Review by LeahBineth — COMMENTED** [review]
Should be fixed now

**Review by mordechaigellis — COMMENTED** [review]
all good

**Pull request closed without merging**